### PR TITLE
Add scalameter-examples version bumping and tagging as release step

### PR DESCRIPTION
**Pull request by kjanosz**

### CODE REVIEW

File: project/ReleaseExtras.scala [hunk 1]
```diff
@@ -65,4 +76,68 @@ object ReleaseExtras {
...
+          case _ => st.log.warn("Remember to push the changes yourself!")
+        }
+      } else {
+        st.log.info(s"Changes were NOT pushed, because no upstream branch is configured for the local branch [${vc.currentBranch}]")
+      }
+    }
+
+    IO.withTemporaryDirectory { tmpDir =>
+      Process("git" :: "clone" :: repo :: "." :: Nil, tmpDir) !! st.log
+      val examplesGit = new Git(tmpDir)
```
Comment (axel22) [inline project/ReleaseExtras.scala]: Could you add some logging output here to track progress.

File: project/Build.scala [hunk 2]
```diff
@@ -35,6 +35,12 @@ object ScalaMeterBuild extends MechaRepoBuild {
 
   val releasePluginSettings =  releaseSettings ++ Seq(
     releaseBranchName := s"version/${(version in ThisBuild).value}",
+    examples.repo := "git@github.com:scalameter/scalameter-examples.git",
+    examples.tag := "v%s",
+    examples.tagComment := "Release %s",
+    examples.commitMessage := "Set ScalaMeter version to %s",
+    examples.scalaMeterVersionFile := "version.sbt",
```
Comment (axel22) [inline project/Build.scala]: Should this be named something like "scalameterVersion.sbt"?
And should the project definitions in `scalameter-examples` also change their `libraryDependencies` setting to use the content in this file?

Comment (axel22) [inline project/Build.scala]: Ah, disregard this comment - I just noticed that you changed the project definitions in the other repo already.

**Comment by kjanosz** [comment]
Fixed.

**Comment by axel22** [comment]
Awesome!
Would you mind adding a short description to document the release process here:

https://github.com/scalameter/scalameter.github.com/blob/master/home/releasing.md

**Pull request merged**